### PR TITLE
Show login required modal on dashboard

### LINKED ISSUE #158: Dashboard UX: Add conditional asking users to login when attempting to add or request review
[Issuehunt badges](https://issuehunt.io/r/PREreview/rapid-prereview/issues/158)

On `outbreaksci.prereview.org/dashboard` when a user is not logged in and clicks the button(s) to Add Review or Request Review, nothing happens. We need to replicate the behavior that's currently on the main homepage, where if a user is logged out and attempts to add a review, a modal pops up and asks the user to log in with their ORCID.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $200.00)

- [prereview prereview](https://issuehunt.io/u/prereview) ($200.00)

### Submitted pull Requests
- [#161 Show login required modal on dashboard](https://issuehunt.io/r/PREreview/rapid-prereview/pull/161)
---

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/PREreview/rapid-prereview/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
</details>

### PR DESCRIPTION
**Pull request by leonardosfl**
Fix #158

I think that someone forgot to add that 😄 

It seems to be small change to be worth $100. If you want, you can change the price :)

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#158: Dashboard UX: Add conditional asking users to login when attempting to add or request review](https://issuehunt.io/repos/208844948/issues/158)
---
</details>

### CODE REVIEW

**Comment by leonardosfl** [comment]
@harumhelmy I had no idea that it was going to be that simple :smile:.

> If you think it's fair to add a couple of more related tasks to this, it would be great if you could also add a similar conditional to the Request | Review buttons on the PreprintCard component on the Dashboard (this behavior is also already on the homepage). Totally understand if you'd rather not, though.

I tested that yesterday and it was working. I'll take a look again

**Comment by leonardosfl** [comment]
@harumhelmy It's working. Is this the expected behavior?
76cbc1bf-0a41-472b-9c26-95f49106df1e

**Comment by harumhelmy** [comment]
Well darn, I could have sworn that wasn't working for me yesterday. 😅 Just ran it again locally and it's working great. Sorry about that! 

That did make me think of another fix I think might be good if you're still down to work on it/for this issue's bounty (again, please only do it if you feel it's fair, you are welcome to yuck this and I'm happy to merge this fix). Could we _not_ redirect users to the homepage when you're adding a new preprint from the dashboard? I haven't looked into it much, might just be a matter of adding a new route or something. Let me know and thank you as always ^_^

**Comment by leonardosfl** [comment]
@leonardosfl Sure. I'd love to help. I'll take a look.

**Comment by leonardosfl** [comment]
@harumhelmy It was quite some work 😄. Everything seems to be working as expected.

File: src/components/preprint-card.js [hunk 1]
```diff
@@ -93,7 +94,7 @@ export default function PreprintCard({
   return (
     <Fragment>
       <div
-        className={classNames('preprint-card', { 'preprint-card--new': isNew })}
+        className={classNames('preprint-card', { 'preprint-card--new': isNew && !isDashboardCard })}
```
Comment (leonardosfl) [inline src/components/preprint-card.js:97]: We need this because the dashboard doesn't use box-shadow for the preprint cards (on hover).

**Comment by leonardosfl** [comment]
@harumhelmy Hi, I've resolved the merge conflict.

**Comment by harumhelmy** [comment]
@leonardosfl Thank you so much for all this work! I haven't had time to thoroughly look and test yet, but will do so tomorrow AM EDT. I also spoke with PREReview leadership to see if we could get you a higher bounty reward because I've asked you so far beyond the initial stated issue here 😅

**Comment by leonardosfl** [comment]
@harumhelmy Thank you! I really appreciate it! 😄

**Pull request merged**